### ISSUE TITLE
MacOS: DYLD_LIBRARY_PATH is required after brew install

### ISSUE THREAD

**Issue by fsat** [lightbend/reactive-cli#50]
Deploying https://github.com/fsat/hello-reactive-tooling into Minikube with the released tooling is completed successfully.

I need to `export DYLD_LIBRARY_PATH=/usr/local/opt/curl/lib` though.

I can confirm this is using CLI version `0.3.3`

```
Felixs-MBP-2:hello-reactive-tooling felixsatyaputra$ rp version
rp (Reactive CLI) 0.3.3
```

**Comment by dbrinegar** [comment]
I thought we were trying to point to our own version of curl libs?  Can we dig a bit more?

```
(targz)$ brew upgrade reactive-cli
==> Upgrading 1 outdated package, with result:
lightbend/tools/reactive-cli 0.3.3
==> Upgrading lightbend/tools/reactive-cli 
==> Downloading https://bintray.com/lightbend/generic/download_file?file_path=reactive-cli-0.3.3-Mac_OS_X-x86_64.zip
==> Downloading from https://dl.bintray.com/lightbend/generic/reactive-cli-0.3.3-Mac_OS_X-x86_64.zip?expiry=1512055835921&signature=KKTy2PUAmnch6JAIslsXfxbPEpaM0q0BU%2FcFHlI3TuG
######################################################################## 100.0%
🍺  /usr/local/Cellar/reactive-cli/0.3.3: 5 files, 10.7MB, built in 4 seconds
david@Davids-MacBook-Pro:~/src/github.com/dbrinegar/reactive-cli
(targz)$ rp version
rp (Reactive CLI) 0.3.3
david@Davids-MacBook-Pro:~/src/github.com/dbrinegar/reactive-cli
(targz)$ otool -L /usr/local/opt/reactive-cli/bin/rp
/usr/local/opt/reactive-cli/bin/rp:
	/usr/local/opt/reactive-cli/lib/libhttpsimple.so (compatibility version 0.0.0, current version 0.0.0)
	/usr/local/opt/re2/lib/libre2.0.dylib (compatibility version 0.0.0, current version 0.0.0)
	/usr/local/opt/reactive-cli/lib/libcurl.4.dylib (compatibility version 7.0.0, current version 9.0.0)
	/usr/lib/libc++.1.dylib (compatibility version 1.0.0, current version 307.5.0)
	/usr/lib/libSystem.B.dylib (compatibility version 1.0.0, current version 1238.60.2)
david@Davids-MacBook-Pro:~/src/github.com/dbrinegar/reactive-cli
(targz)$ otool -L /usr/local/opt/reactive-cli/lib/libcurl.4.dylib
/usr/local/opt/reactive-cli/lib/libcurl.4.dylib:
	/usr/local/opt/reactive-cli/lib/libcurl.4.dylib (compatibility version 10.0.0, current version 10.0.0)
	/usr/local/opt/openssl/lib/libssl.1.0.0.dylib (compatibility version 1.0.0, current version 1.0.0)
	/usr/local/opt/openssl/lib/libcrypto.1.0.0.dylib (compatibility version 1.0.0, current version 1.0.0)
	/System/Library/Frameworks/LDAP.framework/Versions/A/LDAP (compatibility version 1.0.0, current version 2.4.0)
	/usr/lib/libz.1.dylib (compatibility version 1.0.0, current version 1.2.8)
	/usr/lib/libSystem.B.dylib (compatibility version 1.0.0, current version 1238.60.2)
	/System/Library/Frameworks/CoreFoundation.framework/Versions/A/CoreFoundation (compatibility version 150.0.0, current version 1349.8.0)
```

**Comment by dbrinegar** [comment]
curious mainly about those libssl and libcrypto libs.  what do you see there on your system?

**Comment by longshorej** [comment]
> I thought we were trying to point to our own version of curl libs? Can we dig a bit more?

Yeah, that's the intention. Agree that `otool` output would be good. Also curious if `DYLIB_LIBRARY_PATH=/usr/local/opt/reactive-cli/lib:$DYLIB_LIBRARY_PATH` works.

**Comment by fsat** [comment]
Fixed in `0.4.1` via #56
